node/src/Commands.ts

```diff
@@ -2384,6 +2383,12 @@ export type LolwutOptions = {
      *  For version `6`, those are number of columns and number of lines.
      */
     parameters?: number[];
+    /**
+     * An optional argument specifies the type of decoding.
+     *  Use Decoder.String to get the response as a String.
+     *  Use Decoder.Bytes to get the response in a buffer.
+     */
+    decoder?: Decoder;
```

Comment:
Should be added to all options, isn't it?

Reply:
it should be added to all commands which returns string. and it will be gradually. I have now added only the get command. 
I will make more complex commands in the next commit so that others can follow. Once we have done this, we will be able to divide the work

Reply:
Ok thanks